yaksh/test_models.py

```diff
@@ -1553,6 +1554,20 @@ def test_duplicate_attempt_answerpaper(self):
                 course=self.answerpaper.course
                 )
 
+    def test_get_categorized_question_indices(self):
+        all_questions = self.answerpaper.get_all_ordered_questions()
+        categorized_question_indices = \
+            self.answerpaper.get_categorized_question_indices()
+        questions = all_questions[0:4]
+        category_question_map = defaultdict(lambda: [])
+        for index, question in enumerate(questions, 1):
+            category_question_map[
+                    legend_display_types[question.type]["label"]
+                    ].append(index)
+        category_question_map = dict(category_question_map)
```

Comment:
Exactly same code is written as it is in the model method!
Remove this. Simply create a dict with key value expected and compare with the got dict.
Also will be good if there are more test cases for this.